### PR TITLE
Update the man page and the windows installer info

### PR DESCRIPTION
**Pull request by Umcaruje**
The site info and version info was outdated.

### CODE REVIEW

**Comment by Wallacoloo** [comment]
Looks fine to me. I wonder if we can extract the version info dynamically from the git tags to prevent that from getting outdated again.

File: lmms.1 [hunk 1]
```diff
@@ -63,7 +63,7 @@ Show version information and exit.
 .IP "\fB\-h, --help
 Show usage information and exit.
 .SH SEE ALSO
-.BR http://lmms.sf.net/
-.BR http://lmms.sf.net/wiki/
+.BR http://lmms.io/
```
Comment (grejppi) [inline lmms.1]: *https

Comment (Umcaruje) [inline lmms.1]: Fixed it :+1:

**Comment by tresf** [comment]
> I wonder if we can extract the version info dynamically from the git tags to prevent that from getting outdated again.

`.in` files normally do variable substitution automatically, so I would agree with this statement.

**Comment by tresf** [comment]
Actually, the proper sytax is used lower in the file, just copy that `@VERSION` line.


I think the idea here is that the `.lmmsrc.xml` file can be versioned itself.  Matching the software version is probably ideal though.  Might want to allow some convo first, but my vote is to replace it with `@VERSION` like below.

**Comment by Wallacoloo** [comment]
I don't know enough about this file to tell if this is important or not, but changing it from `FILEVERSION	1,0,93,0` to `FILEVERSION @VERSION@,0` would actually yield `FILEVERSION 1.1.3,0` (notice, `.` instead of `,`).

To retain the current format, but having it update dynamically, the proper fix would look something like `FILEVERSION @VERSION_MAJOR@,@VERSION_MINOR@,@VERSION_PATCH@,0`.

These macros are defined in [CMakeLists:16-18](https://github.com/LMMS/lmms/blob/master/CMakeLists.txt#L16-L18).

**Comment by Wallacoloo** [comment]
Also, this file does indeed get variable substitution. See [CMakeLists:371](https://github.com/LMMS/lmms/blob/master/CMakeLists.txt#L371).

**Comment by tresf** [comment]
> the proper fix would look something like

Agreed. :+1:

**Comment by Umcaruje** [comment]
Updated the fileversion line to `FILEVERSION @VERSION_MAJOR@,@VERSION_MINOR@,@VERSION_PATCH@,0`

**Comment by tresf** [comment]
:+1:

**Comment by Umcaruje** [comment]
The travis build passed, is there anything holding this from being merged?

**Comment by tresf** [comment]
It will conflict with #2131, so one or the other will likely need a hand-edit after this is merged.  Merge it now if you like, you just might need to help the OP of #2131 to fix conflicts and squash some commits afterward. :+1:

**Comment by Umcaruje** [comment]
Ok, fair enough. Merging.

**Pull request merged**